common/src/main/java/org/corfudb/common/metrics/micrometer/protocoltransformer/influx/TimerInfluxLineTransformer.java

```diff
@@ -23,6 +23,6 @@
     public boolean test(String s) {
         Matcher matcher = timerPattern.matcher(s);
         this.matched = Optional.of(matcher);
-        return matcher.find();
+        return matcher.matches();
```

Comment:
Can you add tests?

The regex is not correct. The timer outputs a decimal for throughout and not an integer.

The following test will fail
```java

@Test
    public void timerTest() {
        String dataPoint = "rpc-latency{endpoint=localhost:9000} throughput=3.4/s mean=0.000204964s max=0.001044081s";
        TimerInfluxLineTransformer txn = new TimerInfluxLineTransformer();
        assertThat(txn.test(dataPoint)).isTrue();
    }
```

You can fix it with 

```java
protected static final String TIMER_PATTERN_STRING =
            METRIC_PATTERN_STRING + " (throughput=[\\d.]+/s mean=[\\d.]+s max=[\\d.]+s)$";
```

Reply:
Was fixed